experimental/devices/pca9685/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package pca9685 includes utilities to controls pca9685 module and servo motors.
+// More details
+//
+// See https://periph.io/device/pca9685/ for more details about the device.
+//
+// Based on https://github.com/wintersandroid/Android-Things-PCA9685
+//
+// Datasheets:
+// https://cdn-shop.adafruit.com/datasheets/PCA9685.pdf
+//
+// Product page:
+// https://www.adafruit.com/product/815
```

Comment:
The product page is https://www.nxp.com/products/analog/interfaces/ic-bus/ic-led-controllers/16-channel-12-bit-pwm-fm-plus-ic-bus-led-controller:PCA9685

I do link to adafruit on the website in the "where to buy" section but not in the documentation.